spec/spec_helper_acceptance.rb

```diff
@@ -1,26 +1,26 @@
-require 'beaker-puppet'
 require 'beaker-rspec'
+require 'beaker-puppet'
 require 'beaker/puppet_install_helper'
 require 'beaker/module_install_helper'
 
 run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+install_ca_certs unless ENV['PUPPET_INSTALL_TYPE'] =~ %r{pe}i
```

Comment:
Any particular reason to change this file? I think the before suite is actually correct. Also not sure if we need the `install_ca_certs` part.

Reply:
No. I have seen this kind of configuration in our repositories quite often and thought it was some kind of standard.

Reply:
I've been removing them whenever I saw them because the tests also passed without them. https://github.com/puppetlabs/beaker-puppet_install_helper/blob/41c355b1f83fde562dd5bd6961e2e4eb8b81421c/lib/beaker/ca_cert_helper.rb#L13-L31 suggests this is a Windows/Solaris workaround but we don't really test those in most modules.

Reply:
So it actually does nothing, because this module does not support Windows / Solaris. Fixed it, anyway.